fixtures/attribute-behavior/src/App.js

```diff
@@ -764,7 +772,7 @@ class App extends React.Component {
       ReactDOMStable:
         'https://unpkg.com/react-dom@latest/umd/react-dom.development.js',
       ReactDOMServerStable:
-        'https://unpkg.com/react-dom@latest/umd/react-dom-server.browser.development.js',
+        'https://unpkg.com/react-dom@latest/umd/react-dom-server-legacy.browser.development.js',
```

Comment:
What caused this change?

Reply:
`renderToString` is no longer included in `react-dom-server.browser`. Makes sense considering `renderToString` is considered legacy. `renderToString` was already excluded from the [18.0.0 UMD bundle](https://unpkg.com/react-dom@18.0.0/umd/react-dom-server.browser.development.js) so there was no change to the UMD bundle in a SemVer minor.

Reply:
oh odd, I thought that was already the case. sounds good